.env.test

```
@@ -0,0 +1,14 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+TRELLO_API_KEY=fake key
+TRELLO_API_TOKEN=fake api
+TRELLO_BOARD_ID=Fake board
+TRELLO_TODO_LIST_ID=<fake to do
+TRELLO_DONE_LIST_ID=<fake list
```

Not that it matters, but it would be nice to not have the leftover `<` here:
```suggestion
TRELLO_TODO_LIST_ID=fake to do
TRELLO_DONE_LIST_ID=fake list
```